py3status/modules/playerctl.py

```diff
@@ -65,22 +79,30 @@ class Py3status:
     """ """
 
     # available configuration parameters
+    button_cycle_loop_status = None
     button_next = None
     button_pause = None
     button_play = None
     button_play_pause = 1
     button_previous = None
+    button_seek_forward = None
+    button_seek_backward = None
     button_stop = 3
+    button_toggle_shuffle = None
+    button_volume_up = None
+    button_volume_down = None
     format = "{format_player}"
     format_player = (
         r"[\?color=status [\?if=status=Playing > ][\?if=status=Paused \|\| ]"
         r"[\?if=status=Stopped .. ][[{artist}][\?soft  - ][{title}|{player}]]]"
     )
     format_player_separator = " "
     players = []
+    seek_change = 5
     thresholds = {
         "status": [("Playing", "good"), ("Paused", "degraded"), ("Stopped", "bad")]
     }
+    volume_change = 10
```

Comment:
(Suggestion)
Rename to:
- `button_shuffle`
- `seek_delta`
- `volume_delta`
- `button_loop_status` Current naming looks fine following playerctl func too. This is me trying to keep button config short / simple... and to rearrange button configs too.

Honestly, it looks absolutely ready enough. Nice looping on checking position.

I can't remember, but I think the problem with cmus was that it's still using mpris1.

Reply:
What's strange with cmus is that it works when I use the playerctl CLI, but it doesn't when I use the library (the CLI runs the same function as the library though). I filed a bug report on playerctl for this inconsistent behavior, but there hasn't been any activity on the project in almost two years so it might never be looked at.